nmostesting/NMOSTesting.py

```diff
@@ -774,7 +870,7 @@ def main(args):
         TOOL_VERSION = "Unknown"
 
     # Start the DNS server
-    if ENABLE_DNS_SD and DNS_SD_MODE == "unicast":
+    if CONFIG.ENABLE_DNS_SD and CONFIG.DNS_SD_MODE == "unicast":

```

Comment:
Does this and other checks in `main` need to be applied when there are subsequent changes to config?

Reply:
Caveats added to README, so can revisit later if demand